pkg/kubelet/kubelet_pods.go

```diff
@@ -1076,6 +1077,7 @@ func (kl *Kubelet) generateAPIPodStatus(pod *v1.Pod, podStatus *kubecontainer.Po
 func (kl *Kubelet) convertStatusToAPIStatus(pod *v1.Pod, podStatus *kubecontainer.PodStatus) *v1.PodStatus {
 	var apiPodStatus v1.PodStatus
 	apiPodStatus.PodIP = podStatus.IP
+	apiPodStatus.QOSClass = qos.GetPodQOS(pod)
```

Comment:
just add a note that this is needed to ensure upgrade from 1.5 to 1.6 to set the field on upgrade.